packages/calcite-components/src/components/sort-handle/sort-handle.stories.ts

```diff
@@ -0,0 +1,38 @@
+import { html } from "../../../support/formatting";
+
+export default {
+  title: "Components/SortHandle",
+};
+
+export const closed = (): string => html`
+  <calcite-sort-handle label="test" set-position="4" set-size="10"></calcite-sort-handle>
+`;
+
+export const open = (): string =>
+  html`<calcite-sort-handle label="test" set-position="4" set-size="10" open></calcite-sort-handle>`;
+
+export const firstPosition = (): string =>
```

Comment:
Could you merge the all position stories into a single one (similar to [this](https://github.com/Esri/calcite-design-system/blob/dev/packages/calcite-components/src/components/combobox/combobox.stories.ts#L843))?